prow/flagutil/github.go

```diff
@@ -49,7 +50,9 @@ func (o *GitHubOptions) AddFlagsWithoutDefaultGithubTokenPath(fs *flag.FlagSet)
 
 func (o *GitHubOptions) addFlags(wantDefaultGithubTokenPath bool, fs *flag.FlagSet) {
 	o.endpoint = NewStrings("https://api.github.com")
+	o.gitEndpoint = "https://github.com"
 	fs.Var(&o.endpoint, "github-endpoint", "GitHub's API endpoint (may differ for enterprise).")
+	fs.StringVar(&o.gitEndpoint, "git-endpoint", o.gitEndpoint, "GitHub endpoint (may differ for enterprise).")
```

Comment:
@midnightconman - can you add a test for the validation of github options ?

Reply:
Great idea! Done.